docs/install/index.md

```diff
@@ -271,6 +271,41 @@ dependencies {
       </ol>
     </td>
   </tr>
+  <tr>
+    <td>iOS</td>
+    <td>C, C++</td>
+    <td><b> CocoaPods: onnxruntime-training-c </b></td>
+    <td> 
+      <ul>
+        <li> In your CocoaPods <code>Podfile</code>, add the <code>onnxruntime-training-c</code> pod:
+          <pre>
+            use_frameworks!
+            pod 'onnxruntime-training-c' 
```

Comment:
You might need to remove all the indent. Otherwise, it may render incorrectly (with a lot of space before `use_frameworks`.
I think if possible, try to build the webpage and see how it got built.

Reply:
Added preview: 

https://vraspar.github.io/onnxruntime/docs/install/